arch/AArch64/AArch64BaseInfo.c

```diff
@@ -17,7 +17,8 @@
 #ifdef CAPSTONE_HAS_ARM64
 
 #if defined (WIN32) || defined (WIN64) || defined (_WIN32) || defined (_WIN64)
-#pragma warning(disable:4996)
+#pragma warning(disable:4996)			// disable MSVC's warning on strcpy()
+#pragma warning(disable:28719)		// disable MSVC's warning on strcpy()
```

Comment:
It is about strcpy(). 

Precisely, 28719 is a PREfast warning for  a use of "[banned API](https://msdn.microsoft.com/en-us/library/bb288454.aspx)" such as strcpy(), strncpy() etc and not exclusive to strcpy(). For example, we get the same warning against strncpy() in SparcInstPrinter.c, and in this file and XCoreInstPrinter.c, it warns use of strcpy().

Since we already suppressed 4996, which is almost same warning as 28719 I would say, it makes sense to me to suppress 28719 as well. 

Note that something we should a little bit more carefully review is changes in XCoreInstPrinter.c: expanding suppression to an entire file from particular lines.  I made this change to follow other file's style (and I thought the updated form was more readable while new risk was minimal).